### ISSUE TITLE
11 Core Elements Tab - Training

### ISSUE THREAD

**Issue by tfremming1** [IncidentManagementSystems/ims-issue-tracker#177]
This is a new requirement by the State of MN.  How we handle for other states we'll have to cross that bridge.

This page will work largely like ISO.  However there are NO Minimums for hours the page on staff (separate story) will show summary for the fire fighter.

Honestly if you build it exactly like ISO I think we'll be OK.

Topic will be replaced with Core Element
Clicking "Add" will lead to the same screen as ISO except will select Core Elements
Category tag can be removed.  Adding Page will look like below.  I think this is "easiest" as we don't need the actual sub categories, just the top level.

![image.png](https://images.zenhubusercontent.com/5c34b588ea2ab97902b1bfdf/ee822561-626d-455f-8a69-342554fab1e4)

1. Safety and Protective Equipment
a. A culture of a safe working environment
b. 16 life safety initiatives
c. PPE checks
d. Bloodborne pathogens
e. Proper PPE for the work place
2. Chemistry of Fire and Fire Behavior
a. Fire Behavior
b. Building Construction
c. Fire Tactics/Strategies
d. Thermal imager
e. Foam
f. Extinguishers
3. Self-Contained Breathing Apparatus (SCBA)
a. Checks/ recharge SCBA cylinder
b. Donning/doffing
c. PASS devices
d. Use/Care
e. Confidence course
4. Fire Streams
a. Attack below grade
b. Attack ground grade
c. Attack above grade
5. Hose
a. Types
b. Loads/Lays
c. Coupling drill
d. Deployment techniques
6. Pumping Fire Apparatus
a. Responding (CEVO or Emergency Vehicle Operations)
b. Pumping
c. Drafting
d. Supply to appliances
171
10
Revised 02/2015
7. Ladders
a. Ladders
b. Deployment
c. Safety
d. Aerials
8. Rescue
a. Firefighter
i. Search
ii. Self-rescue
iii. Mayday operations
iv. Rapid intervention
b. Extrication
i. Auto recognition
ii. Highway safety
iii. Tools
iv. Airbags
v. Other cutting devices
vi. Disentanglement
vii. Other machinery
viii. Farm equipment or Mining equipment
c. Ropes
i. Rope types
ii. Knots
iii. Use
d. First Aid
i. First Aid
ii. CPR
9. Forcible Entry
a. Tool Identification
b. Forcible entry
i. Door
ii. Window
iii. Wall
iv. Other
10. Ventilation
a. Horizontal
b. Vertical
c. Mechanical
11. Administrative/Command
a. Professional Development
b. Financial Management
c. Human Resources
d. Incident Command System

**Comment by rkiran34** [comment]
I've talked to Travis regarding this - I suggested that we show all core elements with hours in the tab. This avoids the need for the functionality to add elements, which I thought would eventually be added to the core elements tab. So this tab becomes just a read-only view unless we need to give the user the ability to delete these cards.

I'm also contemplating the idea of just showing these in a table view instead of cards. Let me know what you guys think.
